modules/nodes-cluster-limit-ranges-about.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * nodes/cluster/limit-ranges.adoc
+
+[id="nodes-cluster-limit-ranges-about_{context}"]
+= About limit ranges
+
+A limit range, defined by a `*LimitRange*` object, restricts resource 
+consumption in a project. In the project you can set specific resource 
+limits for a Pod, container, image, image stream, or 
```

Comment:
s/image stream/ImageStream/

Reply:
@jboxman This is a reference to a generic image stream, not an ImageStream object, I would think.